### DISCUSSION TITLE
Custom new tab URL

### DISCUSSION THREAD

**Discussion by MatiasDesuu** [Floorp-Projects/Floorp#1463] [Feature Request]
An option to change the new tab and home page URL.

The idea is that when you press the Home button (or create a new tab) the browser redirects to a custom URL (useful if you have a self-hosted Dashboard or maybe you want to set a webpage as your Home page)

This is already possible in a lot of Chromium-based web browser, a good example is Brave, which have the option to set a custom Home Page as well as new tab page

This option is missing in a lot of Firefox-based browsers (as well as in Firefox itself), and is one of the main reasons I have to do some workarounds to switch to Firefox-based browsers.

On desktop, there is a hidden about:config variable to change this (browser.newtab.url) which is disabled by default but can be activated with custom scripts and does exactly what I requesting here :)

**Comment by CutterKnife** [comment]
Floorp has a flag in `about:config` called `floorp.newtab.overrides.newtaburl`.

**Reply by MatiasDesuu** [reply]
Despite it works, there is a problem with that flag, it doesn't clear the URL if you set a custom one, so you need to clear the URL every time you open a new tab and want to search something.

Maybe that's a bug/feature that can be added in the future (or maybe there is any other flag that I don't know that also clears the URL)

**Comment by TotoCodeFR** [comment]
+1, being able to have my homepage setup in my new tab would be great (with a cleared URL ofc)
